_data/merchants.yml

```diff
@@ -209,6 +211,10 @@
       url: http://investmentart.org/
     - name: Mushroom cultures, mushroom growing supplies, seeds
       url: https://www.vesp.co/Fungible
+    - name: Plazmatic
+      url: http://www.plazmatic.com
```

Comment:
Same of above, Monero is not in the accepted forms of payment: https://www.plazmatic.com/pages/faq